packages/calypso-e2e/src/lib/components/domain-search-component.ts

```diff
@@ -0,0 +1,70 @@
+import { Page } from 'playwright';
+
+const selectors = {
+	searchInput: `input[aria-label="What would you like your domain name to be?"]`,
+	resultPlaceholder: `.is-placeholder`,
+	resultItem: ( keyword: string ) => `.domain-suggestion__content:has-text(${ keyword })`,
```

Comment:
Ah, I think you're missing quotes around here! It fails otherwise unfortunately!
```suggestion
	resultItem: ( keyword: string ) => `.domain-suggestion__content:has-text("${ keyword }")`,
```

Reply:
Good catch, thanks. Not sure how that made it through.

Reply:
Fixed in 3c5d75c615396095f70524b6e99748c0ad17f321.